ts/screens/modal/RegionServiceModal.tsx

```diff
@@ -0,0 +1,191 @@
+import CookieManager, { Cookie } from "@react-native-community/cookies";
+import { Body, Container, Content, Left, Text, View } from "native-base";
+import * as React from "react";
+import { Alert, ScrollView, TextInput } from "react-native";
+import { heightPercentageToDP } from "react-native-responsive-screen";
+import WebView, { WebViewMessageEvent } from "react-native-webview";
+import I18n from "../../i18n";
+import ButtonDefaultOpacity from "../../components/ButtonDefaultOpacity";
+import { Monospace } from "../../components/core/typography/Monospace";
+import ActivityIndicator from "../../components/ui/ActivityIndicator";
+import AppHeader from "../../components/ui/AppHeader";
+import IconFont from "../../components/ui/IconFont";
+import customVariables from "../../theme/variables";
+import { getCurrentLocale } from "../../utils/locale";
+import { showToast } from "../../utils/showToast";
+import {
+  APP_EVENT_HANDLER,
+  AVOID_ZOOM_JS,
+  closeInjectedScript
+} from "../../utils/webview";
+
+type Props = {
+  onModalClose: () => void;
+};
+
+const RegionServiceModal: React.FunctionComponent<Props> = (props: Props) => {
+  const [navigationURI, setNavigationUri] = React.useState("");
+  const [webMessage, setWebMessage] = React.useState("");
+  const [text, setText] = React.useState("");
+  const [loading, setLoading] = React.useState(false);
+  const [success, setSuccess] = React.useState(false);
+  const [error, setError] = React.useState(false);
+
+  React.useEffect(() => {
+    const cookie: Cookie = {
+      name: "test",
+      value: "test",
+      domain: "192.168.1.20", // the domain on which you need to add the cookie, in this case the IP on your machine
+      path: "/",
+      version: "1",
+      expires: "2021-05-30T12:30:00.00-05:00"
+    };
+    /**
+     * the domain on which you need to add the cookie, in this case the IP on your machine
+     * this value and domain value on cookie MUST match
+     * this parameter is considered only by Android.
+     */
+    CookieManager.set("http://192.168.1.20", cookie).catch(_ =>
+      showToast(
+        `Unable to execute cookie authentication for ${cookie.name}`,
+        "danger"
+      )
+    );
+
+    if (navigationURI === "") {
+      setNavigationUri("http://192.168.1.20:3000/test-html");
+    }
+  });
+
+  const showSuccessContent = () => (
```

Comment:
Maybe success/error content should be an absolute view over the content (and not in replace of it)
It should looks like a modal alert

<img src="https://i.stack.imgur.com/Hwsxc.png"/>

Reply:
If you follow the mockup of the project the success is displayed as aa full screen component 
https://projects.invisionapp.com/share/XQY82O9YV83#/screens/430167700